### PR TITLE
Update lexicon.js

### PR DESCRIPTION
**Pull request by Cyfek**
Update to lexicon (first part, will do the rest tomorrow)

### CODE REVIEW

**Comment by mr-martian** [comment]
"zisnam" is missing a closing bracket. There's a handful of consistency things in the glosses, but I'll fix them myself later. (apologies for my complete absence of late)

**Comment by Cyfek** [comment]
Oh, no problem. I'm so sure how to fix things like that, so if you could commit it whenever, then fix it, that'd be great

**Comment by Cyfek** [comment]
@mr-martian hey, it's done

File: lexicon.js [hunk 1]
```diff
@@ -115,18 +115,20 @@ var lexicon = {
...
     "goltsut": ["last", "final"],
-    "gulk": ["sick"],
+    "gulk": ["sick", "ill"],
     "jafinmeht": ["melodramatic"],
     "jam": ["new"],
     "jelk": ["full"],
     "jyt": ["spatial"],
     "kak": ["darkened"],
-    "kek": ["supplemental"],
+    "kek": ["supplemental", "more],
```
Comment (mr-martian) [inline lexicon.js:131]: missing close quote

**Review by mr-martian — CHANGES_REQUESTED** [review]

**Pull request merged**